pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -172,6 +172,14 @@ func (c *Cluster) makeDeployment(nodeName string, selection rookalpha.Selection,
 		commonArgs = append(commonArgs, "--default-log-to-file", "false")
 	}
 
+	// OSD fails to find the right IP to bind to when running on SDN
+	// for more details: https://github.com/rook/rook/issues/3140
+	if !c.HostNetwork {
+		if c.clusterInfo.CephVersion.IsAtLeast(version.CephVersion{Major: 14, Minor: 2, Extra: 2}) || c.clusterInfo.CephVersion.IsAtLeastOctopus() {
```

Comment:
@noahdesu did I get this right? I remember that in an old PR you said `IsAtLeast` only works for the current Major. Here I want to cover from 14.2.2 or Octopus, thanks!

Reply:
IIUC I think you only need the first part of this conditional. The `IsAtLeast` case would cover 14.2.2, 14.2.3, ..., 15.x, ... The `IsAtLeastXXX` variant is the same but comparison starts at `X.0.0`. The `IsRelease` only checks the major.

Reply:
Were you trying to do something like allow "if 14.2.2+, OR current master?" the idea being you want to commit NOW but catch the case of different container versions being available at different times?

Reply:
My goal is to catch "as of 14.2.2". Thanks for the clarification.`